fog/view/enclave/impl/src/lib.rs

```diff
@@ -196,11 +196,11 @@ where
     /// to be sent off to each Fog View Store shard.
     fn create_multi_view_store_query_data(
         &self,
-        client_query: EnclaveMessage<ClientSession>,
+        sealed_query: SealedClientMessage,
```

Comment:
Echoing the above comment, if this is a handle to the sealed data, I'd recommend renaming to something like "sealed_query_handle" etc.